dungeon/build.gradle

```
@@ -0,0 +1,73 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+plugins {
+    id 'java-library'
+    id 'antlr'
+}
+
+dependencies {
+    api project(":game")
```

```suggestion
    api project(':game')
```
weiter unten auch... immer ' verwenden, wenn es keine inline vars gibt